liquibase-core/src/main/java/liquibase/integration/commandline/CommandLineResourceAccessor.java

```diff
@@ -34,17 +35,17 @@ public InputStreamList openStreams(String relativeTo, String streamPath) throws
 
 
     @Override
-    public SortedSet<String> list(String relativeTo, String path, boolean includeFiles, boolean includeDirectories, boolean recursive) throws IOException {
-        SortedSet<String> contents = new TreeSet<>();
-        Set<String> superList = super.list(relativeTo, path, includeFiles, includeDirectories, recursive);
+    public List<Resource> find(String relativeTo, String path, boolean includeFiles, boolean includeDirectories, boolean recursive) throws IOException {
```

Comment:
Any reason this is now a list? Seems like a set is appropriate since there wouldn't be duplicate files.

Reply:
I went back and forth on SortedSet vs. List vs. Collection. 

The reason I went away from SortedSet is that I was worried about all the different Resource and ResourceAccessors trying to write implementations of compareTo that works with all the possibly very-different implementations of Resource. If Resource doesn't define methods that can fully differentiate between unique resources, how could we implement a comparator in general?

I went with List over Collection because we do want a defined order, because there is logic that sometimes says "take the first entry" and we want that to be at least vaguely consistent.

Since making it a list, though, I added getDescription() to the Resource interface which could maybe act as that differentiator? It was more targeted at logging and so wasn't expecting people to make sure it's unique and/or makes sense for sorting but we could add that requirement to the method. Or we could add something like getURI as well or in place of getDescription to make it more constrained on what it should be like and therefore more obviously unique?

Reply:
Because Resource has a generic String getDescription() rather than a getURI() I had to update/break the InputStreamList to track Strings as keys rather than URIs. That is probably also a good reason to swicth `Resource.getDescription()` to `getURI()`

Reply:
I ended up realizing it has to be List since it's the resource accessor that needs to decide on the order, not something a comparator. 

I did also replace getDescription() with getURI() to be more forced in what it actually is.